openstack/workflow/v2/executions/requests.go

```diff
@@ -0,0 +1,44 @@
+package executions
+
+import "github.com/gophercloud/gophercloud"
+
+// CreateOptsBuilder allows extension to add additional parameters to the Create request.
+type CreateOptsBuilder interface {
+	ToExecutionCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts specifies parameters used to create an execution.
+type CreateOpts struct {
+	// WorkflowID is the unique id of the workflow.
+	WorkflowID string `json:"workflow_id" required:"true"`
+
+	// WorkflowNamespace is the namespace of the workflow.
+	WorkflowNamespace string `json:"workflow_namespace,omitempty"`
+
+	// Input is a JSON structure containing workflow input values, serialized as string.
+	Input string `json:"input,omitempty"`
+
+	// Params define workflow type specific parameters.
+	Params string `json:"params,omitempty"`
```

Comment:
Params looks to be a map/dict:

https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/execution.py#L277

Reply:
Same as Input and Output :)